Source/WebCore/svg/SVGTransformList.cpp

```diff
@@ -62,54 +62,75 @@ AffineTransform SVGTransformList::concatenate() const
     return result;
 }
 
-template<typename CharacterType> bool SVGTransformList::parseGeneric(StringParsingBuffer<CharacterType>& buffer)
+template<typename CharacterType> bool SVGTransformList::parseGeneric(StringParsingBuffer<CharacterType>& buffer, ListReplacement listReplacement)
```

Comment:
I think `ListReplacement` makes things a little bit more complicated. Why do not we make this function work in the replace mode as long as the type of the `existingTransform` matches the type of  `parsedTransform`? And once the type differ, just call `resize(itemIndex);` and this will delete the remaining of all the `existingTransforms`

Reply:
I don't think that's possible, because some callers call it multiple times and expect the transforms to accumulate.